### DISCUSSION TITLE
Custom Ore Generator Hilfe

### DISCUSSION THREAD

**Discussion by Nik2810-Original** [DerFrZocker/Custom-Ore-Generator#10] [Q&A]
Okay, dann mache ich hier weiter. Wie soll ich die Config senden?

**Comment by DerFrZocker** [comment]
Du kannst die config hier als code block hinzufügen, oder einer der unzähligen paste seiten verwenden: https://paste.md-5.net/, https://www.toptal.com/developers/hastebin, https://hastebin.skyra.pw/, https://pastebin.com/
